docs/how-tos/dbt_extensions.rst

```diff
@@ -1,14 +1,47 @@
 .. _dbt-extensions:
 
-DBT extensions
-**************
-
-To extend the DBT project, you can use the following Tutor variables:
-
-- **DBT_REPOSITORY**: A git repository URL to clone and use as the DBT project.
-- **DBT_BRANCH**: The branch to use when cloning the DBT project.
-- **DBT_PROJECT_DIR**: The directory to use as the DBT project.
-- **EXTRA_DBT_PACKAGES**: A list of python packages for the DBT project to install.
-- **DBT_ENABLE_OVERRIDE**: This variable determines whether the DBT project override feature
-  should be enabled or not. When enabled, it allows you to make changes to the **dbt_project.yml**
-  and **packages.yml** files using the tutor patches: `dbt-packages` and `dbt-project`.
+Extending dbt
+*************
+
+As noted in :ref:`dbt`, you can install your own custom ``dbt`` package to apply your own transforms to the event data
+in Aspects.
+
+**Step 1. Create your dbt package**
+
+See `Building dbt packages`_ for details.
+
+**Step 2. Link to aspects-dbt**
+
+Aspects charts depend on the transforms in `aspects-dbt`_, so it's important that your ``dbt`` package also installs
+`aspects-dbt`_.
+
+To do this, add a ``packages.yml`` file to your ``dbt`` package at the top level, with content like this:
+
+.. code-block:: yaml
+
+  packages:
+    - git: "https://github.com/openedx/aspects-dbt.git"
+      revision: v2.2
+
```

Comment:
Should note here that it will also need to contain all of the contents of the aspects-dbt packages.yml too, I believe.

Reply:
Is it true? I don't think so. I been using it with only the `aspects-dbt` for two other projects and it has shown no issues. I think dependencies are installed recursively

Reply:
@bmtcril I think @Ian2012 is correct here.

I couldn't find docs that verify it, which I would expect to find if dbt dependencies didn't work like other package dependencies.

I also dug through the [dbt packages on dbt hub](https://hub.getdbt.com/) to find an example. This demo project [depends on dbt-codegen](https://github.com/dbt-labs/dbt-cloud-simulation-project-XXXL/blob/main/packages.yml), which in turn [depends on dbt-utils](https://github.com/dbt-labs/dbt-codegen/blob/main/packages.yml), but dbt-utils need not be present in the parent's [packages.yml](https://github.com/dbt-labs/dbt-cloud-simulation-project-XXXL/blob/main/packages.yml).